src/Kernel/package.json

```diff
@@ -1,6 +1,24 @@
 {
+    "scripts": {
+        "test": "jest"
+    },
     "devDependencies": {
         "@types/codemirror": "0.0.56",
-        "@types/node": "^14.0.12"
+        "@types/jest": "^26.0.4",
+        "@types/node": "^14.0.12",
+        "canvas": "^2.6.1",
+        "jest": "^26.1.0",
+        "ts-jest": "^26.1.2",
+        "typescript": "^3.9.6"
+    },
+    "jest": {
+        "transform": {
+            "^.+\\.tsx?$": "ts-jest"
+        },
+        "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.ts$",
```

Comment:
Should `__tests__` be `__test__` here, to match your folder structure? Also should `spec` be `snap`, or is that something different?

Reply:
Good catch! I guess it looks like the second half of the regex matches the incorrectly named folder. Fixing it!
`spec` is another naming convention for test files I believe. I'll remove it since it's not being used.

Reply:
actually, it seems like the [default case](https://jestjs.io/docs/en/configuration#testregex-string--arraystring) already matches my files, so I'll just remove this line